pkg/envoy/lds/ingress.go

```diff
@@ -69,6 +69,7 @@ func (lb *listenerBuilder) newIngressHTTPFilterChain(cfg configurator.Configurat
 func (lb *listenerBuilder) getIngressFilterChains(svc service.MeshService) []*xds_listener.FilterChain {
 	var ingressFilterChains []*xds_listener.FilterChain
 
+	// TODO(steeling): shouldn't this call GetPortToProtocolMappingForService?
```

Comment:
Remove this, we discussed offline why the existing code is correct.

Reply:
yup, good call